### DISCUSSION TITLE
fabric核心服务器

### DISCUSSION THREAD

**Discussion by HMwancheng** [MCSLTeam/MCSL2#106] [提问区域]
主要是没在启动器里找到这个选项，所以想问问有没有加上fabric核心服务器的想法（或者已经有了但是我不认识），纯小白发言，谢作者大大回答~

**Comment by lxhtt** [comment]
不知道在说哪个（悲

**Reply by HMwancheng** [reply]
我的意思是，能直接加fabric MOD的服务器有吗，有的话请问是哪个，没有的话能不能加一个（呜呜呜抱歉叙述实在是不清楚）

**Comment by kitUIN** [comment]
使用fabric核心的服务器部署起来倒是简单,主要是不知道谁家提供镜像
我只知道官方下载地址 https://fabricmc.net/use/server/
